src/routes/Cluster/Node/index.tsx

```diff
@@ -85,30 +96,48 @@ class Node extends React.Component<NodeProps, NodeState> {
     const time = new Date(this.props.nodes[node].detail.createAt * 1000);
     return (
       <div>
-        {this.renderListItemContent(
-          <FormattedMessage id={`node.detail.createdAt`} />,
-          time.toISOString()
-        )}
-        {this.renderListItemContent(
-          <FormattedMessage id={`node.detail.status`} />,
-          this.props.nodes[node].detail.status
-        )}
-        {this.renderListItemContent(
-          <FormattedMessage id={`node.detail.os`} />,
-          this.props.nodes[node].detail.os
-        )}
-        {this.renderListItemContent(
-          <FormattedMessage id={`node.detail.kernelVersion`} />,
-          this.props.nodes[node].detail.kernelVersion
-        )}
-        {this.renderListItemContent(
-          <FormattedMessage id={`node.detail.kubeproxyVersion`} />,
-          this.props.nodes[node].detail.kubeproxyVersion
-        )}
-        {this.renderListItemContent(
-          <FormattedMessage id={`node.detail.kubernetesVersion`} />,
-          this.props.nodes[node].detail.kubernetesVersion
-        )}
+        <Row>
+          <Col span={12}>
+            {this.renderListItemContent(
+              <FormattedMessage id={`node.detail.createdAt`} />,
```

Comment:
if you don't have concat variable, just use plan single quote rather than literal template.

Reply:
and also could remove bracket.